fix Dockerfile: dont fetch source code

moeryomenko opened the pull request:
Just copy from repo.

Signed-off-by: Maxim Eryomenko <moeryomenko@gmail.com>

moeryomenko committed:
fix Dockerfile: dont fetch source code

Just copy from repo.

Signed-off-by: Maxim Eryomenko <moeryomenko@gmail.com>

EremenkoVO commented on .dockerignore:
```diff
@@ -0,0 +1,2 @@
+.git
+yarn.lock
```
yarn.lock is need

moeryomenko replied:
for what?

EremenkoVO replied:
for true tree dependencies

moeryomenko replied:
think this no matter

EremenkoVO commented on Dockerfile:
```diff
@@ -1,13 +1,12 @@
-    && yarn install \
+COPY . .
+
+RUN yarn install \
```
just 'yarn'

moeryomenko committed:
simplify build script

Signed-off-by: Maxim Eryomenko <moeryomenko@gmail.com>

moeryomenko committed:
remove yarn.lock from .dockerignore

https://stackoverflow.com/questions/39990017/should-i-commit-the-yarn-lock-file-and-what-is-it-for

Signed-off-by: Maxim Eryomenko <moeryomenko@gmail.com>

moeryomenko committed:
Docker: separate app build and run stages

Dive report current total image size: 123 MB

Signed-off-by: Maxim Eryomenko <moeryomenko@gmail.com>
